"too many arguments to function ‘tmx_get_tile’" causes build to fail

I don't know if this game is still being maintained, but I'd at least like to be able to build it.

First I installed the tmx library you're using (all of the output is included.
```
$ git clone git://github.com/baylej/tmx.git; cd tmx
$ mkdir build; cd build
$ cmake ..
-- The C compiler identification is GNU 4.9.2
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Configuring done
-- Generating done
-- Build files have been written to: /home/kaashif/src/tmx/build
$ make
Scanning dependencies of target tmx
[ 20%] Building C object CMakeFiles/tmx.dir/src/tmx.c.o
[ 40%] Building C object CMakeFiles/tmx.dir/src/tmx_utils.c.o
[ 60%] Building C object CMakeFiles/tmx.dir/src/tmx_err.c.o
[ 80%] Building C object CMakeFiles/tmx.dir/src/tmx_xml.c.o
[100%] Building C object CMakeFiles/tmx.dir/src/tmx_json.c.o
Linking C static library libtmx.a
[100%] Built target tmx
$ sudo make install
[100%] Built target tmx
Install the project...
-- Install configuration: ""
-- Installing: /usr/local/lib/libtmx.a
-- Installing: /usr/local/include/tmx.h
```

Then I tried to build this project:
```
$ make
cc -Wall -Werror -std=c99 -DASSETS='"assets/"' -c -g src/entity.c
cc -Wall -Werror -std=c99 -DASSETS='"assets/"' -c -g src/game.c
cc -Wall -Werror -std=c99 -DASSETS='"assets/"' -c -g src/animation.c
cc -Wall -Werror -std=c99 -DASSETS='"assets/"' -c -g src/main.c
cc -Wall -Werror -std=c99 -DASSETS='"assets/"' -c -g src/util.c
cc -Wall -Werror -std=c99 -DASSETS='"assets/"' -c -g src/collision.c
cc -Wall -Werror -std=c99 -DASSETS='"assets/"' -c -g src/render.c
src/render.c: In function ‘drawLayer’:
src/render.c:227:18: error: too many arguments to function ‘tmx_get_tile’
             ts = tmx_get_tile(map, layer->content.gids[(i*map->width)+j],
                  ^
In file included from src/render.c:2:0:
/usr/local/include/tmx.h:141:11: note: declared here
 tmx_tile* tmx_get_tile(tmx_map *map, unsigned int gid);
           ^
src/render.c:227:16: error: assignment from incompatible pointer type [-Werror]
             ts = tmx_get_tile(map, layer->content.gids[(i*map->width)+j],
                ^
cc1: all warnings being treated as errors
Makefile:16: recipe for target 'render.o' failed
make: *** [render.o] Error 1
```

If it works for you, then maybe you're using a different version of libtmx to the latest one? Maybe there's something else I'm doing wrong? 

I'm on Debian testing, with all of the dependency libraries and SDL 2 up to date.

--- a/src/render.c
+++ b/src/render.c
@@ -224,7 +224,7 @@ void drawLayer(SDL_Renderer *ren, SDL_Texture *res, tmx_map *map, tmx_layer *lay
 
     for (i=0; i<map->height; i++) {
         for (j=0; j<map->width; j++) {
-            ts = tmx_get_tile(map, layer->content.gids[(i*map->width)+j],
+            ts = tmx_get_tileset(map, layer->content.gids[(i*map->width)+j],
                               // FIXME: This probably aint good way to go..
                               (unsigned int *)&(srcrect.x),
                               (unsigned int *)&(srcrect.y));

